tests/cases/fourslash/getEmitOutputTsxFile_ReactNative.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path="fourslash.ts" />

```

Comment:
The other compiler testcases I made after this also test the PR's behaviour, so I suppose I can just delete this fourslash one.